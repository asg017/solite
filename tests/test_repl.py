import re

def test_repl(solite_cli, snapshot):
    output = solite_cli([], communicate=[b".timer off\nselect 1 + 1;\n"], kill=True).stdout
    output = re.sub(r"Solite \d+\.\d+\.\d+(-[a-z]+\.\d+)?", "Solite VERSION", output)
    assert output == snapshot
