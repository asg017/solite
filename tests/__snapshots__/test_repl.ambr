# serializer version: 1
# name: test_repl
  '''
  Solite VERSION (SQLite 3.47.0)
  Enter ".help" for usage hints.
  Connected to a transient in-memory database.
  ┌───────┐
  │ 1 + 1 │
  ├───────┤
  │     2 │
  └───────┘
  ^D
  
  '''
# ---
